### PR TITLE
Add new config properties to Docker setup

### PR DESCRIPTION
**Pull request by wederbn**
#### Short Description
Add new config properties to Docker setup

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -24,10 +24,14 @@ ENV CONTAINER_REPOSITORY_CONTEXT winery
 ENV CONTAINER_BUS_MANAGEMENT_MOCK false
 ENV CONTAINER_DEPLOYMENT_TESTS false
 
-ENV ENGINE_IA_HOSTNAME localhost
-ENV ENGINE_IA_PORT 8090
-ENV ENGINE_IA_USER admin
-ENV ENGINE_IA_PASSWORD admin
+ENV ENGINE_IA_JAVA8_HOSTNAME localhost
```
Comment (lharzenetter) [inline Dockerfile:27]: I wouldn't change the existing ones to avoid breaking the current docker-compose setup

**Review by lharzenetter — APPROVED** [review]

**Comment by lharzenetter** [comment]
relates to https://github.com/OpenTOSCA/opentosca-docker/pull/39/files

**Comment by lharzenetter** [comment]
is required by https://github.com/OpenTOSCA/tosca-definitions-common/pull/18

**Review by nyuuyn — APPROVED** [review]
lgtm

**Pull request merged**